### PR TITLE
feature:readme title

### PR DESCRIPTION
**Pull request by dakt**
Please pull from me

### CODE REVIEW

**Comment by dakt** [comment]
Anyone?

**Comment by darijo-alma** [comment]
Hello

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# alma-test
\ No newline at end of file
+#Hello World
```
Comment (darijo-alma) [inline README.md:1]: Why Hello World? Doesn't the original have ","?

Comment (dakt) [reply README.md:1]: Yes, but ever since Java people are leaving out comma

Comment (darijo-alma) [reply README.md:1]: Ok, I guess we'll do the same

Comment (darijo-alma) [reply README.md:1]: Before that, we should check if water is still liqud

**Review by darijo-alma — APPROVED** [review]
Lookin good now

**Pull request merged**